server/dotenv/test.env

```diff
@@ -1,5 +1,5 @@
 JWT_SECRET=dooboolab
-DATABASE_URL="postgresql://postgres:dooboolab0!@localhost:5432/postgres?schema=test"
```

Comment:
Changing this will not pass the `circleci` config.

Reply:
according to this `DATABASE_URL`, here's the values
- user: **postgres**
- password: **dooboolab@!**
- url: **localhost**
- database: **postgres**
- scheme: **test**

compared with `default server setting` section,
```
CREATE DATABASE hackatalk;
CREATE ROLE postgres WITH LOGIN NOSUPERUSER INHERIT CREATEDB NOCREATEROLE NOREPLICATION PASSWORD 'dooboolab0!';
GRANT CONNECT ON DATABASE hackatalk TO postgres;
GRANT ALL PRIVILEGES ON DATABASE hackatalk TO postgres;
```

I think following actions to be done.
- `CREATE DATABASE postgres`
- `CREATE SCHEMA test` // after connecting 'postgres' database
  - (maybe it doesn't need to create because it will be dropped after "yarn test")

User doesn't need to be added because user(ROLE) `postgres` is already added in previous step.

Is that right?

Reply:
True. This was more simple approach since user don't have to create another `database user` for testing.